source/manual/access-to-licensing-for-third-parties.html.md

```diff
@@ -21,74 +21,131 @@ The source code is hosted on GitHub at [alphagov/licensify](https://github.com/a
 
 Licensify uses an Elasticsearch / Logstash / Kibana system hosted by Logit.io for its logs.
 
-If you haven't already got access to GDS' Logit account, you'll need to [follow the instructions in the Reliability Engineering documentation to create an account in Logit](https://reliability-engineering.cloudapps.digital/logging.html#get-started-with-logit).
+If you haven't already got access to the GDS LogIt account, you'll need to [follow the instructions in the Reliability Engineering documentation to create an account in Logit](https://reliability-engineering.cloudapps.digital/logging.html#get-started-with-logit).
 
-## Accessing the VPN
+## Creating new Builds
 
-This is a prerequisite for accessing Jenkins (which is used for deployments), and for SSH-ing onto instances.
+Licensify is built into Docker container images with Github Actions Workflows.
 
-Follow [the VPN guidance for non-GDS devices ("BYOD")](https://docs.google.com/document/d/150JX1xiWdXY29ahcYUMb05Si-hEAZvtkGAKojT9Rjis/edit)
-to set up the VPN. You will need to sign into your `@digital.cabinet-office.gov.uk` Google account to access this document.
+If you have access to merge code into main, you will have access to start builds that, once completed, will push container images into AWS ECR (Elastic Container Registry) and then trigger ArgoCD to start a deployment.
```

Comment:
Might be worth adding here - instruction on how to manually deploy via GitHub Actions.

Also explaining that every merge onto main automatically gets built and deployed to integration. But they must manually deploy the application to staging and production.

Reply:
Added a paragraph about deploy triggers.